.yarnrc.yml

```diff
@@ -17,13 +17,23 @@ yarnPath: .yarn/releases/yarn-3.3.0.cjs
 
 packageExtensions:
   "@craco/craco@*":
+    dependencies:
+      # postcss is included already in the `react-scripts` package
+      postcss: "^8.4.4"
     peerDependencies:
       "@types/node": "*"
       typescript: "*"
   "@craco/types@*":
     dependencies:
-      postcss: "^8.1.0"
-      eslint: "^7.0.0 || ^8.0.0"
+      postcss: "^8.4.4"
+      eslint: "^8.3.0"
```

Comment:
same here
@a-ignatov-parc  I have taken the versions from `react-scripts` package. Should I replace the versions with `"*"` ?
```
      postcss: "*"
      eslint: "*"
```

Reply:
@r-bartolomei, please do the same as I explained in